server/src/main/java/org/spine3/server/event/Subscribe.java

```diff
@@ -28,12 +31,17 @@
 /**
  * Marks a method as an event subscriber.
  *
- * <p>An event subscriber method must have two parameters. The class of the event will be
- * indicated by the first parameter.
+ * <p>An event subscriber/handler method:
```

Comment:
Yes, it would be OK because Event Subscriber is a special kind of Handler Method.